docs-site/src/markdown-pages/design/technical-design.md

```diff
@@ -0,0 +1,175 @@
+
+- **Loosely coupled** - To enable EOP to change and expand in the future
+  decisions now that promote loose coupling between components. An example of
+  trying to reduce coupling is the use of Kafka as the integration hub. This
+  allows for the components to be changed out without affecting the other
+  components.
+
+- **Scalable** - EOP is going to start small and grow ... so we need tech that
+  supports that. To enable EOP to grow to support the needs of the region. This
+  means that the solution needs to be able to scale up and down as needed. This
+  includes the ability to scale up and down the number of regions that are
+  supported by EOP.
+
+- **Extensible** - To enable EOP to be able to integrate with other systems and
+  services. This includes the ability to integrate with other systems of record
+  and other systems that provide services that EOP can use.
+
+## Structure
+
+EOP is being developed as a hub and spoke model. Data is captured or acquired
+from external sources and sent to the hub in a raw format. That raw format can
+then be forwarded to different storage or processing components depending on how
+that data will be used, for example, it might be stored in a data warehouse for
+analytical analysis or to a real time processing component for operational
+alerting. Key to this model is not forcing all data into a store which would
+limit its to be used for different use cases.
+
+A decision has been made to build the hub on [Kafka](https://kafka.apache.org/).
+Kafka was chosen to provide a scalable, loosely coupled, hub with well supported
+connectors to other systems. While Kafka is often used for real time processing
+it is also well suited to consuming data that may come from batch processes from
+the councils.
+
+The following diagram shows the high-level relationship between the different
+classes of components that will be delivered around the hub
+
+![High Level Overview Diagram](./high-level-overview.png)
+
+### Acquisition
+
+Acquisition components support the process of getting data captured in existing
+systems and into EOP. These might be existing council systems or any third party
+systems which EOP can make use of. Acquiring data in this fashion will be the
+main focus of the initial EOP build.
+
+Generally, the preference is for councils to push data into EOP rather than
+pull. This allows councils to choose the frequency of updates from their systems
+to EOP. However, there will be some cases where pulling data from a system is
+required, for example from third party systems that can't be changed.
+
+A special case is Hilltop which because of its ubiquity in councils and it
+having a pre-existing API will be supported as a "pull from councils" example.
+
+Example Acquisition components:
+
+- JSON API's exposed to councils
+- Data lake / Blob storage style end points for bulk data
+- Hilltop API
+- Direct connection from Kafka Connectors running in council environments
```

Comment:
```suggestion
Acquisition components support ingestion of data from existing systems into EOP.
These might be existing council systems or third party systems housing data
which EOP can make use of. Acquisition will be the the initial focus for EOP.

The preference is for councils to push data to EOP, rather than for EOP to pull.
This allows councils to control the frequency of updates. However, there will be
some cases where pulling data from a system is required, such as data from third
party systems that can't be changed.

A special case is Hilltop. Because of its wide-use and existing APIs, EOP will
support pulling data from council Hilltop instances.

Example Acquisition components:

- JSON API's exposed to councils
- Data lake / Blob storage style end points for bulk data
- Hilltop API
- Direct connection from Kafka Connectors running in council environments
```